src/components/Typography/Typography.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Typography from './Typography';
+
+describe('Typography', () => {
+  it('renders', () => {
+    render(<Typography data-testid="Typography-Test" />);
+    expect(screen.getByTestId('Typography-Test')).toBeInTheDocument();
+  });
+
+  it('renders the appropriate tag', () => {
```

Comment:
One more nice to have test (and story)...
You could verify that it renders an arbitrary React component in the `tag` props.